core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2977,6 +2977,14 @@ private[spark] object Utils extends Logging {
       0
     }
   }
+
+  /**
+   * Returns blockmanager heartbeat timeout as milliseconds.
+   */
+  def blockManagerHeartbeatTimeoutAsMs(conf: SparkConf): Long = {
```

Comment:
I think you don't have to bother with this util and test. You can just write down, in the PR description,  that you tested it like this locally

Reply:
OK